### PR TITLE
neovim-gtk: Add v0.1.1_545

### PR DESCRIPTION
**Pull request by alexschneider**

### CODE REVIEW

File: bucket/neovim-gtk.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://github.com/daa84/neovim-gtk",
+    "version": "0.1.1_545",
+    "description": "GTK ui for neovim written in rust using gtk-rs bindings. With ligatures support.",
+    "license": {
+        "identifier": "GPL3",
```
Comment (rasa) [inline bucket/neovim-gtk.json]:
```suggestion
        "identifier": "GPL-3.0-only",
```
Per https://spdx.org/licenses/

File: bucket/neovim-gtk.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://github.com/daa84/neovim-gtk",
+    "version": "0.1.1_545",
+    "description": "GTK ui for neovim written in rust using gtk-rs bindings. With ligatures support.",
```
Comment (rasa) [inline bucket/neovim-gtk.json]:
```suggestion
    "description": "A highly configurable text editor for efficiently creating and changing any kind of text. (Neovim/GTK version, with ligatures support)",
```
Let's re-use the description for vim, OK?

File: bucket/neovim-gtk.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        "neovim-gtk\\dist\\bin\\nvim-gtk.exe"
+    ],
+    "shortcuts": [
+        [
+            "neovim-gtk\\dist\\bin\\nvim-gtk.exe",
+            "Neovim GTK"
+        ]
+    ],
+    "suggest": {
+        "neovim": "extras/neovim"
```
Comment (rasa) [inline bucket/neovim-gtk.json]:
```suggestion
        "neovim": "neovim"
```
Neovim is in the main bucket: https://github.com/ScoopInstaller/Main/blob/master/bucket/neovim.json

File: bucket/neovim-gtk.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+        "identifier": "GPL3",
+        "url": "https://github.com/daa84/neovim-gtk/blob/master/LICENSE"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://ci.appveyor.com/api/buildjobs/lj7awg3qbi99hjp4/artifacts/nvim-gtk-mingw64.7z",
+            "hash": "E7DEF6E21C48AF2DD613081B69B49318C41112CD036AA33B465B0D2F7571BE60"
+        }
+    },
+    "extract_to": "neovim-gtk",
```
Comment (rasa) [inline bucket/neovim-gtk.json]: I don't think `extract_to` is needed here.

**Comment by rashil2000** [comment]
Their last release was in 2019, they don't have release binaries, and the AppVeyor artifact has expired

**Comment by rashil2000** [comment]
I am closing this because the project is inactive and there's nothing to download.

**Pull request closed without merging**